resources/dicts/thoughts/alive/kitten.json

```diff
@@ -59,11 +59,11 @@
         "id": "kit_older_cats",
         "thoughts": [
             "Is asking older cats how kits are born",
-            "Wishes other cats would stop babying them",
+            "Wishes other cats would stop babying {PRONOUN/m_c/subject}",
```

Comment:
you've got some 'subject' in places where they should be 'object', 'them' is an object not a subject